app/dts/behaviors/mod_tap.dtsi

```diff
@@ -1,9 +1,12 @@
 / {
 	behaviors {
 		mt: behavior_mod_tap {
```

Comment:
So... defining mod-tap this way won't allow for setting multiple modifiers to be used for the "hold" behavior here. We can probably get around this by later adding a new `&mods` that just takes a bit-flag set of modifiers, but this is a "regression" to a certain degree.

I have no *personal* need for multiple modifiers for a mod-tap, but I'm not sure if others have a sense of the need for this?

Reply:
Something like `&mods` would be a great idea. I prefer it over requiring a behavior definition for every tap-hold.

I think we should limit the use of behavior definitions for common use cases. A first time user who wants to set up hold-taps should not have to define a behavior for every key, but just call something like `&ht SFT A`. The next step is to change their settings to their preferences by using `&ht { settings = <value> ; }`, and once they want even more configurability (e.g. per-key tapping term) they'll have to write their own behavior. This way, there is an easy way to use this functionality for new users, and a more complex way for power users to configure the keyboard to their liking.